packages/pigeon/platform_tests/flutter_null_safe_unit_tests/lib/core_tests.gen.dart

```diff
@@ -172,16 +172,16 @@ class HostEverything {
         message: 'Host platform returned null value for non-null return value.',
       );
     } else {
-      return (replyMap['result'] as Everything?)!;
+      return (replyMap['result'] as AllTypes?)!;
```

Comment:
There have been some significant changes to there files in my open pr, hopefully handling these conflicts doesn't cause too many new issues.